css/layout.css

```diff
@@ -3,6 +3,7 @@
     height: 65px;
     top: 0;
     left: 0;
+    z-index: 4;
```

Comment:
Try to avoid z-indexes if you can -- it's likely this was not needed ;)

Reply:
Wihout z-index the carousel element go above header element.

Reply:
Ok, just keep the z-indexes in mind -- it gets way too easy to lose track of them. Personally, I'd keep a list of variables like this:

```
--z-index-header: 1000;
--z-index-dropdown: 2000;
--z-index-tooltip: 3000;
// etc
```